composer_class.rb

```diff
@@ -0,0 +1,21 @@
+class Composer
+  attr_accessor :first_name, :last_name, :year_born, :year_died, :age
+  @@all = []
+  @@selected = []
+  def initialize(first_name, last_name, year_born, year_died)
+    @first_name = first_name
+    @last_name = last_name
+    @year_born = year_born
+    @year_died = year_died
+    @age = year_died.to_i - year_born.to_i
+    @@all.push(self)
+    @@selected.push(self)
+  end
+  def self.all
+    @@all
+  end
+  def self.selected
+    @@selected
+  end
```

Comment:
Add a method to select the composer instance that can be used later on (after the instance has been created)

```suggestion
  end
  def select
    @@selected.push(self)
  end
```